rfcs/056-prismic-etl-pipeline/README.md

```diff
@@ -0,0 +1,45 @@
+# RFC 056: Prismic to Elasticsearch ETL pipeline
+
+**Status:** Draft
+
+**Last updated:** 23/2/2023
+
+## Context
+In order to make our editorial content - including stories, comics, exhibitions and events - more discoverable, we want to be able to search it via an API as described in [RFC 055](https://github.com/wellcomecollection/docs/tree/main/rfcs/055-prismic-api).
+
+While Prismic does provide some search functionality of its own (which we have been using for initial versions of unified search), we want more control and fewer limitations. To achieve this we want to use Elasticsearch, as we do for our other search services. As such, we need to get data from Prismic into Elasticsearch: this RFC will propose the mechanism by which we achieve that.
+
+#### Desiderata
+- Changes (including additions and edits but possibly not deletions - see BEEPB OPOSDF) in Prismic are reflected promptly in the Elasticsearch index
```

Comment:
> See BEEPB OPOSDF

I thought this read [OSPF](https://en.wikipedia.org/wiki/Open_Shortest_Path_First) (the routing algorithm) because I am old. 👵 

What do you actually mean here?